sequelizeModels/Province.js

```diff
@@ -0,0 +1,10 @@
+class Province{
```

Comment:
Delete this file. This is from the library output.
Also the ouput should be located inside the folder who executed the library, in this case is ./example/sequelize-app